### PR TITLE
vimPlugins.{substrata-nvim,vim-substrata,vim-printer}

### PR DESCRIPTION
**Pull request by alexisquintero**
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by alexisquintero** [comment]
Result of `nixpkgs-review pr 179929` run on x86_64-linux

<details>
  <summary>168 packages built:</summary>
  
  spacevim vimPlugins.LeaderF vimPlugins.SchemaStore-nvim vimPlugins.SpaceVim vimPlugins.TrueZen-nvim vimPlugins.aerial-nvim vimPlugins.airline vimPlugins.ale vimPlugins.barbar-nvim vimPlugins.bufferline-nvim vimPlugins.calendar vimPlugins.catppuccin-nvim vimPlugins.catppuccin-vim vimPlugins.chad vimPlugins.cmp-clippy vimPlugins.cmp-copilot vimPlugins.cmp-dap vimPlugins.cmp-dictionary vimPlugins.cmp-digraphs vimPlugins.cmp-fish vimPlugins.cmp-fuzzy-buffer vimPlugins.cmp-fuzzy-path vimPlugins.cmp-git vimPlugins.cmp-greek vimPlugins.cmp-look vimPlugins.cmp-neosnippet vimPlugins.cmp-npm vimPlugins.cmp-nvim-lsp-signature-help vimPlugins.cmp-nvim-tags vimPlugins.cmp-pandoc-nvim vimPlugins.cmp-path vimPlugins.cmp-rg vimPlugins.cmp-snippy vimPlugins.cmp-tabnine vimPlugins.cmp-tmux vimPlugins.cmp-vim-lsp vimPlugins.cmp-vimwiki-tags vimPlugins.cmp-zsh vimPlugins.coc-lua vimPlugins.comment-nvim vimPlugins.completion-treesitter vimPlugins.conjure vimPlugins.coq-artifacts vimPlugins.coq-thirdparty vimPlugins.coq_nvim vimPlugins.crates-nvim vimPlugins.dashboard-nvim vimPlugins.deoplete-lsp
</details>

File: pkgs/applications/editors/vim/plugins/vim-plugin-names [hunk 1]
```diff
@@ -581,6 +581,7 @@ https://github.com/luukvbaal/stabilize.nvim/,,
 https://github.com/eigenfoo/stan-vim/,,
 https://github.com/darfink/starsearch.vim/,,
 https://github.com/teto/stylish.nvim/,HEAD,
+https://github.com/kvrohit/substrata.nvim/,HEAD,
```
[the remark this answers is not in the document]
Comment (alexisquintero) [reply pkgs/applications/editors/vim/plugins/vim-plugin-names:584]: I just run the script, do you think I should remove it?

[the remark this answers is not in the document]
Comment (superherointj) [reply pkgs/applications/editors/vim/plugins/vim-plugin-names:584]: I don't know. I'm not that much used to this. Just trying to understand. From what I understood it's just a branch. If the script added itself, let's leave it.

**Review by superherointj — APPROVED** [review]

**Comment by alexisquintero** [comment]
It is one package per commit though, do you mean I should have squashed the update commits?
(vim-substrata and substrata-nvim are 2 "different" packages/plugins)

**Comment by superherointj** [comment]
> It is one package per commit though, do you mean I should have squashed the update commits? (vim-substrata and substrata-nvim are 2 "different" packages/plugins)

Squashing the update commits would have been good. But, if it makes more sense, the better anyway. No worry.

**Pull request merged**